.github/workflows/reusable-build-coverage.yml

```diff
@@ -11,6 +11,9 @@ on:
         required: false
         default: 'ubuntu-latest'
         type: string
+    secrets:
+      CODECOV_TOKEN:
+        required: true
```

Comment:
Maybe this is not required if we use `secrets: inherit`?

> Note: If the secrets are inherited by using `secrets: inherit` in the calling workflow, you can reference them even if they are not explicitly defined in the on key.

https://docs.github.com/en/actions/using-workflows/reusing-workflows#using-inputs-and-secrets-in-a-reusable-workflow

Reply:
I am not sure whether this is then also unnecessary on the reusable workflow. I'm currently merely testing things, hence the draft state. I'm having the impression that things aren't sorted out with codecov's v4 action, though. With this version I'm having different trouble `Commit creating failed: ["Service not found: none"]` which seems to pop up in https://github.com/codecov/feedback/issues/263. There seems to be a workaround, but I expect that runs us into https://github.com/codecov/codecov-action/issues/1287. There seems to be work around this, e.g. https://github.com/codecov/codecov-action/commit/f62c5eebfe9df5cfe9d56fd86ec925442fe3c214.

We might consider going back to v3 until things are sorted out?

Reply:
I suggested the same in the comment below ;)